packages/core/src/exporter.ts

```
@@ -245,138 +172,147 @@ export const fillValue = async (
-    case "createdAt":
-      value = moment(value).format("YYYY-MM-DD");
+    case 'createdAt':
+      value = moment(value).format('YYYY-MM-DD');
       break;
 
-    case "branches":
+    case 'branches':
       const branches = await models.Branches.find({
-        _id: item.branchIds
+        _id: item.branchIds,
       }).lean();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)